apps/next-evans/components/core/PillDropdownItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import cx from 'classnames';
+
+const PillDropdownItem: React.FC<{
+  selected?: boolean;
+  className?: string;
+  children: React.ReactNode;
+  onClick?: (e: React.MouseEvent) => any;
+}> = ({ children, className, selected, onClick }) => (
+  <div
+    onClick={onClick}
+    className={cx(
+      className,
+      `PillDropdownItem p-3 select-none`,
```

Comment:
this first class should be unnecessary if we've eliminated all of the standard CSS.